M2/Macaulay2/packages/NormalToricVarieties/Divisors.m2

```diff
@@ -177,7 +177,7 @@ nefGenerators NormalToricVariety := Matrix => X -> (
     fromPic := matrix fromPicToCl X;
     indexOfPic := abs lcm (minors( rank source fromPic, fromPic^torsionlessCoord))_*;
     (indexOfPic * coneGens) // fromPic 
-    );
+    ))
```

Comment:
By definition, the nef cone lives in the Neron--Severi group (which for a toric variety is canonical isomorphic to the Picard group).  Does returning something in the class group make mathematical sense?

Reply:
Sorry, I don't follow. The only change here is caching of `nefGenerators`. What change are you referring to?